test/rtos_drivers/usb/README.rst

```diff
@@ -43,7 +50,9 @@ run the following command from the root of this repository:
 
 .. code-block:: console
 
-    $ ./tools/ci/build_rtos_tests.sh
+    bash tools/ci/build_rtos_tests.sh
+
+The ``build_rtos_tests.sh``` script will copy the test applications to the ``dist`` folder.
```

Comment:
triple quote

Reply:
Fixed in https://github.com/xmos/fwk_rtos/pull/214/files.